# HG changeset patch
# User Rik <rik@octave.org>
# Date 1449168388 28800
#      Thu Dec 03 10:46:28 2015 -0800
# Node ID d0991cbd61416a141d59fe410ba74e459e767a0f
# Parent  f7084eae3318ddfd669099adc8e57b7ae3482cee
maint: Remove extra spaces in if () conditionals.

* gl2ps-renderer.cc, symbfact.cc, ov-base.cc,
file-stat.h: Remove extra spaces in if () conditionals.

diff --git a/libinterp/corefcn/gl2ps-renderer.cc b/libinterp/corefcn/gl2ps-renderer.cc
--- a/libinterp/corefcn/gl2ps-renderer.cc
+++ b/libinterp/corefcn/gl2ps-renderer.cc
@@ -450,17 +450,17 @@ glps_renderer::render_text (const std::s
   std::string str = txt;
   std::list<ft_render::ft_string> lst;
 
   text_to_strlist (str, lst, bbox, ha, va, rotation);
 
   // When using "tex" or when the string has only one line and no
   // special characters, use gl2ps for alignment
   if (lst.empty () || term.find ("tex") != std::string::npos
-      || (lst.size () == 1  && ! lst.front ().get_code ()))
+      || (lst.size () == 1 && ! lst.front ().get_code ()))
     {
       std::string name = fontname;
       int sz = fontsize;
       if (! lst.empty () && term.find ("tex") == std::string::npos)
         {
           ft_render::ft_string s = lst.front ();
           name = select_font (s.get_name (), s.get_weight () == "bold",
                               s.get_angle () == "italic");
diff --git a/libinterp/dldfcn/symbfact.cc b/libinterp/dldfcn/symbfact.cc
--- a/libinterp/dldfcn/symbfact.cc
+++ b/libinterp/dldfcn/symbfact.cc
@@ -94,17 +94,17 @@ The elimination tree itself.\n\
 A sparse boolean matrix whose structure is that of the Cholesky\n\
 factorization as determined by @var{typ}.\n\
 @end table\n\
 @end deftypefn")
 {
   octave_value_list retval;
   int nargin = args.length ();
 
-  if (nargin < 1  || nargin > 3 || nargout > 5)
+  if (nargin < 1 || nargin > 3 || nargout > 5)
     print_usage ();
 
 #ifdef HAVE_CHOLMOD
 
   cholmod_common Common;
   cholmod_common *cm = &Common;
   CHOLMOD_NAME(start) (cm);
 
diff --git a/libinterp/octave-value/ov-base.cc b/libinterp/octave-value/ov-base.cc
--- a/libinterp/octave-value/ov-base.cc
+++ b/libinterp/octave-value/ov-base.cc
@@ -436,17 +436,17 @@ void
 octave_base_value::print_with_name (std::ostream& output_buf,
                                     const std::string& name,
                                     bool print_padding)
 {
   bool pad_after = print_name_tag (output_buf, name);
 
   print (output_buf);
 
-  if (print_padding  && pad_after && ! Vcompact_format)
+  if (print_padding && pad_after && ! Vcompact_format)
     newline (output_buf);
 }
 
 void
 octave_base_value::print_info (std::ostream& os,
                                const std::string& /* prefix */) const
 {
   os << "no info for type: " << type_name () << "\n";
diff --git a/liboctave/system/file-stat.h b/liboctave/system/file-stat.h
--- a/liboctave/system/file-stat.h
+++ b/liboctave/system/file-stat.h
@@ -230,17 +230,17 @@ public:
   void get_stats (bool force = false)
   {
     if (! initialized || force)
       update_internal (force);
   }
 
   void get_stats (const std::string& n, bool force = false)
   {
-    if (n != file_name || ! initialized  || force)
+    if (n != file_name || ! initialized || force)
       {
         initialized = false;
 
         file_name = n;
 
         update_internal (force);
       }
   }
@@ -288,17 +288,17 @@ public:
   void get_stats (bool force = false)
   {
     if (! initialized || force)
       update_internal (force);
   }
 
   void get_stats (int n, bool force = false)
   {
-    if (n != fid || ! initialized  || force)
+    if (n != fid || ! initialized || force)
       {
         initialized = false;
 
         fid = n;
 
         update_internal (force);
       }
   }
